docs/admin/projects.rst

```diff
@@ -437,12 +437,30 @@ to be escaped as ``[[]`` or ``[]]``.
 Screenshot file mask
 ++++++++++++++++++++
 
-Mask of files to allow discovery/update of screenshots, including path from vcs repository.
-It should include one "*" replacing screenshot file name.
+This feature allows the discovery and updating of screenshots through screenshot file masks, using paths from the VCS repository.
+This operates at the component level and necessitates the use of an asterisk "*" to replace the screenshot file name.
 
 Allowed formats are JPEG, PNG, APNG and GIF.
 
-For example ``docs/*.png``.
+Note:
+
+1. The filemask and screenshot filemask are not related. Configure them separatly.
+2. It is a manual job to link a discovered screenshot in a component to a specific translation key.
+
+For example:
+
+Let's assume your VCS repository has a structure like this:
+
+* repository_root/component_A/docs
+
+  * image1.png
+  * image2.jpg
```

Comment:
This is what we use at https://weblate--9434.org.readthedocs.build/en/9434/admin/customize.html#creating-a-python-module:

```suggestion
.. code-block:: text

    component_A
    └── docs
        ├── image1.png
        └── image2.jpg

```